var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -135,6 +135,12 @@ class Qt(Package):
         working_dir="qtwebsockets",
         when="@5.14: %gcc@11:",
     )
+    patch(
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed

?